src/components/PageHero.tsx

```diff
@@ -56,15 +56,17 @@ const PageHero: React.FC<IProps> = ({
       >
         <Box
           maxW={{ base: "full", lg: "container.sm" }}
-          py={{ base: 16, lg: 32 }}
+          pt={{ base: 0, lg: 32 }}
+          pb={{ base: 8, lg: 32 }}
           pl={{ base: 0, lg: 8 }}
-          mr={4}
+          mr={{ base: 0, lg: 4 }}
         >
           <Heading
             as="h1"
             textTransform="uppercase"
             fontSize="md"
             fontWeight="normal"
+            mt={{ base: "0px", lg: 8 }}
```

Comment:
For some reason, `base: 0` wasn't changing the base style at all here.

Reply:
Tried again local... same thing in all browsers 🤷‍♂️. Will update this PR and try in the preview deploy.